libcontainer/cgroups/fs/memory.go

```diff
@@ -108,28 +121,28 @@ func setMemoryAndSwap(path string, cgroup *configs.Cgroup) error {
 		// for memory and swap memory, so it won't fail because the new
 		// value and the old value don't fit kernel's validation.
 		if memoryUsage.Limit < uint64(cgroup.Resources.MemorySwap) {
```

Comment:
```
runc update test_update --memory-swap -1
 [ "$status" -eq 0 ]
```
 This test should cover what you just said, doesn't it? I just added it the test suite and it passed!